js/src/components/enhanced-conversion-tracking-settings/enable-button.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import { __ } from '@wordpress/i18n';
+import { external as externalIcon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { ENHANCED_ADS_CONVERSION_STATUS } from '.~/constants';
+import { useAppDispatch } from '.~/data';
+import AppButton from '.~/components/app-button';
+import useGoogleAdsEnhancedConversionSettingsURL from '.~/hooks/useGoogleAdsEnhancedConversionSettingsURL';
+
+const EnableButton = ( { onEnable = noop } ) => {
+	const {
+		updateEnhancedAdsConversionStatus,
+		updateEnhancedConversionsSkipConfirmation,
+	} = useAppDispatch();
+	const url = useGoogleAdsEnhancedConversionSettingsURL();
+
+	const handleOnEnableEnhancedConversions = () => {
+		window.open( url, '_blank' );
+		updateEnhancedAdsConversionStatus(
+			ENHANCED_ADS_CONVERSION_STATUS.PENDING
+		);
+		updateEnhancedConversionsSkipConfirmation( true );
```

Comment:
This is only needed for the modal, so probably better to pass this in as part of the `onEnable` handler from the modal, rather than explicitly setting this here.

Reply:
👍 The `updateEnhancedConversionsSkipConfirmation` action has been moved. I've kept `updateEnhancedAdsConversionStatus` since it's needed when the CTA is rendered within the Settings panel.